CODE_GENERATION.md

```diff
@@ -50,8 +50,14 @@ When you re-run the generation with this configuration, existing files won't be
 deleted. So you may want to delete everything in `apis/<serviceid>/v1alpha1` except
 `generator-config.yaml` and then re-run the command.
 
-> If `apis/<serviceid>` is created from scratch, please add the new service name
-> to the list called GENERATED_SERVICES in Makefile.
+## Makefile
```

Comment:
```suggestion
### Makefile
```
One more suggestion. This is still part of the "Code Generation" section.

Looks good!

Reply:
I just suggested that header be removed, but this looks fine as well 👍🏻 Choose whichever you prefer @haarchri :)

Reply:
added # =) looks better with header in my point of view 
thanks